docs/getting-started.md

```diff
@@ -31,6 +32,19 @@ Run
 
 Once this has completed you will have a functioning flux cluster.
 
+## Getting an API Token
+
+We use a combination of Ory [Hydra](https://www.ory.sh/docs/hydra) and Ory [Oathkeeper](https://www.ory.sh/docs/oathkeeper) for creating and storing OAuth2 access tokens which are in turn used to access the various APIs used in the dscp project.
```

Comment:
We should probably mention dependencies here

Reply:
It is mentioned on line 16.

Reply:
🤦‍♂️